v2/pkg/genruntime/well_known-resource-reference.go

```diff
@@ -10,7 +10,7 @@ package genruntime
 type WellKnownResourceReference struct {
```

Comment:
This file is not correctly named still (has both underscores and dashes).

Reply:
Renamed.